File: Daily Challanges/2026-04-April/2026-04-09.py
def get_next_bingo_number(n):
    b_num = int(n[1:])% 75 + 1
    if b_num < 16:
        return f"B{b_num}"
    if b_num < 31:
        return f"I{b_num}"
    if b_num < 46:
        return f"N{b_num}"
    if b_num < 61:
        return f"G{b_num}"
    if b_num < 75:
        return f"O{b_num}"
    return "Invalid"

print(get_next_bingo_number("B10"))
print(get_next_bingo_number("O75"))

File: Daily Challanges/2026-04-April/2026-04-09.js
function getNextBingoNumber(n) {
  let bNum = (Number(n.slice(1)) % 75) + 1;
  if (bNum < 16) return `B${bNum}`;
  else if (bNum < 31) return `I${bNum}`;
  else if (bNum < 46) return `N${bNum}`;
  else if (bNum < 61) return `G${bNum}`;
  else return `O${bNum}`;
}

console.log(getNextBingoNumber("B10"));
